src/sourmash/search.py

```diff
@@ -466,7 +480,11 @@ def __next__(self):
 ###
 
 PrefetchResult = namedtuple('PrefetchResult',
-                            'intersect_bp, jaccard, max_containment, f_query_match, f_match_query, match, match_filename, match_name, match_md5, match_bp, query, query_filename, query_name, query_md5, query_bp')
+                            ['intersect_bp', 'jaccard', 'max_containment', 'f_query_match',
+                            'f_match_query', 'match', 'match_filename', 'match_name',
+                            'match_md5', 'match_bp', 'query', 'query_filename', 'query_name',
+                            'query_md5', 'query_bp', 'ksize', 'moltype', 'num', 'scaled',
```

Comment:
num will always be 0 for prefetch, since prefetch requires scaled... right? or am I missing something? :)

Reply:
yes - was just trying to standardize a bit. Functionally not going to matter after #1955, but I can take it out here if you like.

Reply:
oh, ok, it's fine for it to stay in then :)

Reply:
removed in https://github.com/sourmash-bio/sourmash/pull/1952/commits/644a4dd83eb7f6f48ad11c54122ddeecb35a9faa